src/mkdocstrings/handlers/__init__.py

```
@@ -0,0 +1,242 @@
+        in `self` to use as a fallback theme.
+
+        Arguments:
+            directory: The name of the directory containing the themes for this renderer.
+            theme: The name of theme to use.
+        """
+        themes_dir = Path(__file__).parent.parent / "templates" / directory
+        theme_dir = themes_dir / theme
+        if not theme_dir.exists():
+            if hasattr(self, "FALLBACK_THEME"):
+                log.warning(
+                    f"mkdocstrings.handlers: No '{theme}' theme in '{directory}', "
+                    f"falling back to theme '{self.FALLBACK_THEME}'"
+                )
+                theme_dir = themes_dir / self.FALLBACK_THEME
+            else:
+                raise ThemeNotSupported(theme)
+
+        self.env = Environment(autoescape=True, loader=FileSystemLoader(theme_dir))
+        self.env.filters["highlight"] = do_highlight
+        self.env.filters["any"] = do_any
+
+    def render(self, data: DataType, config: dict) -> str:
+        """
+        Render a template using provided data and configuration options.
+
+        Arguments:
+            data: The collected data to render.
+            config: The rendering options.
+
+        Returns:
+            The renderer template as HTML.
+        """
+        raise NotImplementedError
+
+    def update_env(self, md: Markdown, config: dict) -> None:
+        """
+        Update the Jinja environment.
+
+        Arguments:
+            md: The Markdown instance. Useful to add functions able to convert Markdown into the environment filters.
+            config: Configuration options for `mkdocs` and `mkdocstrings`, read from `mkdocs.yml`. See the source code
+              of [mkdocstrings.plugin.MkdocstringsPlugin.on_config][] to see what's in this dictionary.
+        """
+        md = Markdown(extensions=config["mdx"], extensions_configs=config["mdx_configs"])
```

I'm not sure what's going on here, but `Markdown` doesn't accept an `extensions_configs` parameter; it gets silently ignored. `extension_configs` is a thing, though.

https://github.com/Python-Markdown/markdown/blob/5891418956d08ea3ddcbf65281e6882c7e3d9662/markdown/core.py#L96

---

It definitely looks like a typo, and that should really be fixed!